src/cuter.erl

```diff
@@ -362,3 +381,7 @@ suppress_unsupported_mfas(Options) ->
 -spec type_normalization([option()]) -> boolean().
 type_normalization(Options) ->
   not lists:member(?NO_TYPE_NORMALIZATION, Options).
+
+-spec report_metrics([option()]) -> boolean().
+report_metrics(Options) ->
+  lists:member(?REPORT_METRICS, Options).
```

Comment:
`lists:member` works here, but perhaps we should consider making the `Options` a property list at some point (in a separate refactoring step) and use the `proplists` functions instead.

Reply:
ack.